### DISCUSSION TITLE
中文的标点符号应该怎么打比较标准？

### DISCUSSION THREAD

**Discussion by xskxzr** [tuna/thuthesis#601] [Q&A] [answered]
比如破折号，是用`---`还是`——`；引号是用\`\`''还是`""`？

**Comment by Harry-Chen** [comment, marked as answer]
一般直接使用中文的全角标点，你的两个例子，破折号用 `——`，引号用 `“”`

**Comment by zepinglee** [comment]
中文的破折号用“`——`”，引号用“`“”`”，西文用“`---`”，西文用“``''”。
